test/csuite/random_abort/main.c

```diff
@@ -162,15 +162,11 @@ thread_run(void *arg)
 
     testutil_check(td->conn->open_session(td->conn, NULL, NULL, &session));
 
-#if 0
     /*
      * Make sure that alternative threads operate on column-store table
```

Comment:
Can change to single line comment format.

Reply:
Fixed